src/modules/sensors/sensor_params.c

```diff
@@ -1088,6 +1088,16 @@ PARAM_DEFINE_INT32(SENS_EN_SF0X, 0);
 PARAM_DEFINE_INT32(SENS_EN_MB12XX, 0);
 
 /**
+ * TeraRanger Evo (trevo)
```

Comment:
Since this is a TeraRanger sensor, I think it would be better to have a parameter `SENS_EN_TR` and add the option to set to 0, if none, 1 if TeraRanger One, 2 if TeraRanger Evo.